configs/eslint-config-compass/index.js

```diff
@@ -1,143 +1,62 @@
-const path = require('path');
-
-const jsConfigurations = ['eslint:recommended'];
+'use strict';
 
-const tsConfigurations = [
-  ...jsConfigurations,
-  'plugin:@typescript-eslint/recommended',
-  'plugin:@typescript-eslint/recommended-requiring-type-checking',
-];
+const path = require('path');
+const shared = require('@mongodb-js/eslint-config-devtools');
+const common = require('@mongodb-js/eslint-config-devtools/common');
```

Comment:
This seems to have prettier, can we disable these for compass? They are very distracting and don't bring any value (we have precommit hooks to autofix them)

Reply:
oops

Reply:
Ok I'll have to make the change to devtools-shared. Don't know how I missed that.

Reply:
actually.. we should probably just make devtools-shared work like compass does and remove it from there as well.